libbeat/common/uuid.go

```diff
@@ -0,0 +1,22 @@
+package common
+
+import (
+	"crypto/rand"
+	"fmt"
+)
+
+// GenerateUUID is used to generate a random UUID
+// This was copied from https://github.com/hashicorp/uuid/blob/master/uuid.go
```

Comment:
+1. I meant host and time as name spacing to reduce or eliminate the chances of conflicts.